src/modes/simple_select.js

```diff
@@ -233,16 +234,82 @@ module.exports = function(ctx, options = {}) {
       }
     },
     render: function(geojson, push) {
-      geojson.properties.active = (ctx.store.isSelected(geojson.properties.id) )
+      geojson.properties.active = (ctx.store.isSelected(geojson.properties.id))
         ? Constants.activeStates.ACTIVE
         : Constants.activeStates.INACTIVE;
       push(geojson);
       if (geojson.properties.active !== Constants.activeStates.ACTIVE
         || geojson.geometry.type === Constants.geojsonTypes.POINT) return;
       createSupplementaryPoints(geojson).forEach(push);
     },
-    trash() {
+    trash: function() {
       ctx.store.delete(ctx.store.getSelectedIds());
+    },
+    combineFeatures: function() {
+      var selectedFeatures = ctx.store.getSelected();
+
+      if(!selectedFeatures || selectedFeatures.length < 2) return;
+
+      var featureType = selectedFeatures[0].type.replace('Multi','');
+      var coordinates = [];
+      var properties = selectedFeatures[0].properties;
+
+      selectedFeatures = selectedFeatures.map(function(feature) {
```

Comment:
To be clear, this map is doing two things.

1. It is creating the coordinates array for the new MultiFeature
2. Getting the GeoJSON of the selected features that have the same non-multi type of the first feature